blog-service/2023-09-08-search.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Query Timer Enhancement (Search)
+image: https://www.sumologic.com/img/logo.svg
+keywords:
+  - search
+  - query
+  - elapsed timer
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+Update - We've enhanced the elapsed query timer so that now everytime you switch the sumo internal tab and come back the timer does not starts again from 0, instead, it shows the steady elapsed time.
```

Comment:
Do we also need to highlight the issue where our clock run very slow when browser is minimized or some other window in focused instead of the browser? This is because https://usefulangle.com/web-updates/post/110/chrome-88-will-throttle-timers-1-minute

Reply:
I would look to reword this as ----

We have identified an issue with the elapsed query timer, in scenarios where a user switches between SUMO internal tabs when a query is in progress. This bug resulted in inaccurate elapsed query time calculations. With this fix, the elapsed query time will be updated accurately.

Reply:
@manashar4 Usually, in release note we will not mention the cause of the issue instead we just mention the solution and how it is going to help the user. Also, the content above have some passive voice issue. I have updated the content with more clarity now, can you please check.

Reply:
@JV0812 - I feel that makes sense for Features but with Bugs i dont know if that is a good approach. 
In any case, just one minor edit suggestion

We've enhanced the elapsed query timer so that while a query is running and _you_ **users** navigate to a different Sumo internal tab or browser tab and then return, the timer no longer resets to zero seconds but instead displays the precise amount of time that has passed.

Other than that looks good to me.

Reply:
Thanks @manashar4. I have updated this, let me ask for a quick approval on this PR from my team.